justfile

```diff
@@ -99,7 +99,7 @@ update-favicon:
 
 # run go commands in the context of the api project
 go *ARGS:
-  @cd cmd/api/src && GODEBUG=cgocheck=2 go {{ARGS}}
+  @cd cmd/api/src && go {{ARGS}}
```

Comment:
Was this change intentional? It seems out of scope from the original issue.

Reply:
It is out of scope, but that flag is deprecated and keeps just go * commands from running so I just fixed it while testing